src/routes/api/users.js

```diff
@@ -1,28 +1,28 @@
 import express from "express";
 
-const router = express.Router();
+import generateToken from "../../helper/helper.js";
 
+const router = express.Router();
 router.get("/user", (req, res, next) => {
-  res.status(200).json({
-    message: "successfully sent query"
+  res.json({
+    message: "successfully sent query",
   });
 });
-
 router.put("/user", (req, res, next) => {
   res.status(200).json({
-    message: "successfully sent"
+    message: "successfully sent",
   });
 });
 
 router.post("/user/login", (req, res, next) => {
-  res.status(200).json({
-    message: "successfully sent"
-  });
+  const user = { name: "meandyou", password: "hdfjjfvbnj" };
+  const token = generateToken(user);
+  res.json({ token, message: "successfully sent query" });
```

Comment:
Can you please make the suggested change as ignoring it might confuse the one working on login
```suggestion

```

Reply:
Yeah, this will be removed